src/modules/logger/logger.cpp

```diff
@@ -1312,43 +1331,75 @@ void Logger::debug_print_buffer(uint32_t &total_bytes, hrt_abstime &timer_start)
 #endif /* DBGPRINT */
 }
 
-bool Logger::check_arming_state(int vehicle_status_sub, MissionLogType mission_log_type)
+bool Logger::start_stop_logging(int vehicle_status_sub, int manual_control_sp_sub, MissionLogType mission_log_type)
 {
-	bool vehicle_status_updated;
-	int ret = orb_check(vehicle_status_sub, &vehicle_status_updated);
 	bool bret = false;
+	bool want_start = false;
+	bool want_stop = false;
+
+	if (_log_mode == LogMode::rc_aux1) {
+		//aux1-based logging
+		bool manual_control_setpoint_updated;
+		int ret = orb_check(manual_control_sp_sub, &manual_control_setpoint_updated);
+
+		if (ret == 0 && manual_control_setpoint_updated) {
+			manual_control_setpoint_s manual_sp;
+			orb_copy(ORB_ID(manual_control_setpoint), manual_control_sp_sub, &manual_sp);
+			bool should_start = manual_sp.aux1 > 0.3f || _arm_override;
```

Comment:
What about renaming `_arm_override` to `_manually_logging_on`?

Reply:
Changed to `_manually_logging_override`